### ISSUE TITLE
grid = go incompatible with no axes

### ISSUE THREAD

**Issue by dalcde** [hoodmane/spectralsequences#9]
The following snippet:
```latex
\documentclass{article}
\usepackage{spectralsequences}
\begin{document}
\begin{sseqpage}[grid = go, no axes]
  \class(0,0)
\end{sseqpage}
\end{document}
```
results in the following non-fatal error:
```
./test.tex:8: Undefined control sequence.
\sseq@bottomgridpadding ->\sseqxaxisgap

l.8 \end{sseqpage}

The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.

./test.tex:8: Missing number, treated as zero.
<to be read again>
                   <
l.8 \end{sseqpage}

A number should have been here; I inserted `0'.
(If you can't figure out why I needed to see a number,
look up `weird error' in the index to The TeXbook.)

./test.tex:8: Illegal unit of measure (pt inserted).
<to be read again>
                   <
l.8 \end{sseqpage}

Dimensions can be in units of em, ex, in, pt, pc,
cm, mm, dd, cc, nd, nc, bp, or sp; but yours is a new one!
I'll assume that you meant to say pt, for printer's points.
To recover gracefully from this error, it's best to
delete the erroneous units; e.g., type `2' to delete
two letters. (See Chapter 27 of The TeXbook.)

./test.tex:8: Undefined control sequence.
\sseq@bottomgridpadding ->\sseqxaxisgap

l.8 \end{sseqpage}

The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.

./test.tex:8: Missing number, treated as zero.
<to be read again>
                   p
l.8 \end{sseqpage}

A number should have been here; I inserted `0'.
(If you can't figure out why I needed to see a number,
look up `weird error' in the index to The TeXbook.)
```

**Comment by dalcde** [comment]
Adding `\def\sseqxaxisgap{0pt}` to the chart fixes the problem

**Comment by hoodmane** [comment]
Resolved by 354c70835cdd13ff2ed354a39b0d5bf7628f122e.
